geospatial/src/Dem.cc

```diff
@@ -175,9 +201,11 @@ double Dem::Elevation(double _x, double _y)
 {
   if (_x >= this->Width() || _y >= this->Height())
   {
-    gzthrow("Illegal coordinates. You are asking for the elevation in (" <<
-          _x << "," << _y << ") but the terrain is [" << this->Width() <<
-           " x " << this->Height() << "]\n");
+    throw std::invalid_argument(
```

Comment:
We should consider getting rid of these exceptions and printing errors instead. We don't throw exceptions often.

Reply:
I've updated it to an error instead and am returning `inf` if reached. What do you think? 6f6769aedb2ca1689d3a08f295036edec461c099